libr/anal/esil.c

```diff
@@ -424,6 +430,18 @@ R_API bool r_anal_esil_pushnum(RAnalEsil *esil, ut64 num) {
 	return r_anal_esil_push (esil, str);
 }
 
+R_API bool r_anal_esil_pushbig(RAnalEsil *esil, ut8 *num, ut32 size) {
+	RNumBig *big_num = r_big_new();
+	memcpy(big_num, num, size % R_BIG_ARRAY_SIZE);
+	return r_anal_esil_push (esil, r_big_to_hexstr(big_num));
+}
+
+// these numbers are jacked
+R_API RNumBig* r_anal_esil_get_big(RAnalEsil *esil, const char *str, RNumBig *big_num) {
+	r_big_from_hexstr(big_num, str);
+	return big_num;
+}
+
 R_API bool r_anal_esil_push(RAnalEsil *esil, const char *str) {
 	if (!str || !esil || !*str || esil->stackptr > (esil->stacksize - 1)) {
```

Comment:
use R_STR_ISEMPTY (str) and maybe use r_return_val_if_fail () for esil && R_STR_ISEMPTY (str), and do dynamic if for the stacksize comparision

Reply:
this isn't from my PR but I can make these changes

Reply:
So its ready to merge?